packages/lesswrong/server/type3.ts

```diff
@@ -0,0 +1,95 @@
+import { postGetPageUrl } from "@/lib/collections/posts/helpers";
+import { DatabaseServerSetting } from "./databaseSettings";
+import { Globals } from "@/lib/vulcan-lib";
+import { Posts } from "@/lib/collections/posts";
+import { captureEvent } from "@/lib/analyticsEvents";
+
+/* Currently unused
+const type3ClientIdSetting = new DatabaseServerSetting<string | null>('type3.clientId', null)
+const type3WebhookSecretSetting = new DatabaseServerSetting<string | null>('type3.webhookSecret', null)
+*/
+const type3ApiTokenSetting = new DatabaseServerSetting<string | null>("type3.apiToken", null);
+const type3SourceUrlSetting = new DatabaseServerSetting<string>("type3.sourceUrl", "");
```

Comment:
For future notice:

- `type3.apiToken` is a new API token from type3
- `type3.sourceUrl` is set to `https://forum.effectivealtruism.org` (this is needed in order to build absolute prod URLs from a dev machine running the script)